### PR TITLE
Updates Governance

### PR DESCRIPTION
**Pull request by danehans**
Removes Fidelity from governance since no one from the affiliation is contributing to EG.

Signed-off-by: danehans <daneyonhansen@gmail.com>

### CODE REVIEW

**Comment by danehans** [comment]
cc: @richarddli @zinuga

**Review by mattklein123 — APPROVED** [review]

**Comment by richarddli** [comment]
LGTM.

**Comment by zinuga** [comment]
@danehans dont merge this yet. They are clearing their internal hurdles and have just gotten approvals to be able to contribute. It takes a while in bigger companies but they have it now.

**Pull request closed without merging**
